parsing/pcoq.mli

```diff
@@ -40,11 +40,13 @@ module Lookahead : sig
   val contiguous : (int -> Loc.t) -> int -> int -> bool
```

Comment:
Isn't this dead code now?

Reply:
Indeed, I removed it from the interface.